### PR TITLE
Switched to a docker CI build

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
This cuts the build time by half!!

**Comment by LaurentGoderre** [comment]
@nschonni opinion?

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,19 @@
-language: node_js
-node_js:
-  - "6"
-
-sudo: false
+language: generic
+dist: trusty
```
Comment (nschonni) [inline .travis.yml]: From the docs, it sounds like turning on the `docker` under services is the prefered method https://docs.travis-ci.com/user/docker/

Comment (LaurentGoderre) [inline .travis.yml]: That is for the older image. The trusty one dont need it and they are much smaller and faster

**Comment by LaurentGoderre** [comment]
Can we get this merged?

**Review by lucas-hay — APPROVED** [review]

**Pull request merged**